### ISSUE TITLE
capitalized errors ignoring

### ISSUE THREAD

**Issue by grechkin-pogrebnyakov** [golangci/golangci-lint#99]
`warning: error strings should not be capitalized or end with punctuation or a newline (golint)` were skipped with options `--max-issues-per-linter 0 --max-same-issues 0`

**Comment by jirfag** [comment]
thank you!
```
~/go/src/github.com/golangci/golangci-lint (support/analyze-as-much-dirs-as-can-in-case-of-import-errors *)$ gometalinter --disable-all -E golint cap.go
~/go/src/github.com/golangci/golangci-lint (support/analyze-as-much-dirs-as-can-in-case-of-import-errors *)$ golint cap.go
~/go/src/github.com/golangci/golangci-lint (support/analyze-as-much-dirs-as-can-in-case-of-import-errors *)$ golint -min_confidence 0 cap.go
cap.go:1:1: should have a package comment, unless it's in another file for this package
cap.go:11:20: error strings should not be capitalized or end with punctuation or a newline
~/go/src/github.com/golangci/golangci-lint (support/analyze-as-much-dirs-as-can-in-case-of-import-errors *)$ cat cap.go
package main

import "fmt"

func main() {
        err := f()
        fmt.Println(err)
}

func f() error {
        return fmt.Errorf("Capitalized error text")
}
~/go/src/github.com/golangci/golangci-lint (support/analyze-as-much-dirs-as-can-in-case-of-import-errors *)$ golangci-lint run cap.go
cap.go:11:20: error strings should not be capitalized or end with punctuation or a newline (golint)
        return fmt.Errorf("Capitalized error text")
                          ^
~/go/src/github.com/golangci/golangci-lint (support/analyze-as-much-dirs-as-can-in-case-of-import-errors *)$ fgrep -A1 golint .golangci.yml
  golint:
    min-confidence: 0
```

Golint reports this issue sometims with confidence 0.6, modify it in `.golangci.yml`:
```yaml
  golint:
    min-confidence: 0
```

P.S.: [.golangci.example.yml](https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml) now contains description and default value of almost all options
